examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,105 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are pulled from an H5 file and are fit to extract the cross relaxivity of the sample. The R1(p) is first interpolated by fitting the self relaxivity to a polynomial. This fit is then used to fit the linear regime of the enhancement data. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+
+target_directory = os.path.normpath(getDATADIR("AG_processed_data"))
+h5_folder = "ras.h5"
+filename = "220616_E37"
+# {{{Dataset Parameters
+SL_conc_M = 455.2223e-6
+ppt = 1.5167e-3
+Ep_pts = 18
+# }}}
+# {{{ plotting fn
+def list_symbs(f):
+    # {{{ this is just to show all the parameters
+    list_symbs = []
+    for j, k in f.output().items():
+        s_repr = latex(Symbol(j))
+        list_symbs.append(f"${s_repr} = {k:0.5g}$")
+    list_symbs = "\n".join(list_symbs)
+    # }}}
+    return list_symbs
+
+
+# }}}
+with figlist_var() as fl:
+    # {{{ load data
+    Ep = nddata_hdf5(f"{h5_folder}/{filename}/Ep", directory=target_directory)
+    T1p = nddata_hdf5(f"{h5_folder}/{filename}/T1p", directory=target_directory)
+    # }}}
+    # {{{Plot Ep
+    fl.next("E(p)")
+    fl.plot(Ep["power", :-3], "o", label="Experimental Data", capsize=6, alpha=0.5)
+    fl.plot(Ep["power", -3:], "rx", label="Returning Power Check", capsize=6, alpha=0.5)
+    plt.ylabel("E(p)")
+    # }}}
+    # {{{ plot T1p data
+    R1p = 1 / T1p
+    fl.next(r"$R_{1}(p)$")
+    fl.plot(R1p, "o", label="Experimental Data")
+    # }}}
+    # {{{Fit R1p
+    T100 = 3.71
+    dT10 = 0.2
+    a, b, c, power = symbols("a b c power", real=True)  # symbols
```

Comment:
Why not just called p?

Reply:
"power" is a little more clear to the user that that's the axis we are fitting, p doesn't give much information but changed the code anyway

Reply:
Didn’t